standards/SPIFFE_Federation.md

```diff
@@ -307,6 +307,56 @@ As described in the [Trustworthiness of the Bundle Endpoint Server](#74-trustwor
+
+Claims in the third category are not standarized, so operators need to consider them on a case-by-case basis, possibly including navigating different meanings by different trust domains. Needless to say, this category should be approached with the highest degree of caution.
+
+#### 7.7.2 Claim scope
+
+Some claims, like the SPIFFE ID are _explicitly_ scoped to a trust domain by their structure. Because SVIDs are always issued by a particular trust domain authority, _all_ the claims therein are _implicitly_ scoped to that trust domain. This scoping should be preserved when presenting claims to any decision-making process.
+
+As a hypothetical example, consider a claim about the version number of the control plane software that generated the SVID. Different trust domain control planes might use entirely different software packages, and as such the version numbers are not directly comparable in an unscoped context.
+
+
+#### 7.7.3 Interpretation of claims
+
+If claims are to be processed in making a security decision (such as whether to allow a request or connection), it is critical that the authority issuing the claim has the same meaning or interpretation of the claim as the entity making the decision.
+
+In considering the categories of claims noted in [Section 7.7.1](#771-svid-claims), these should be treated with an increasing level of scrutiny around interpretation.
+
+Also consider exactly who is operating the trust domain making the claim. Trust domain operators may be able to agree on the meaning of a particular claim. For example, if an organization operates multiple trust domains for administrative reasons, it may be straightforward to agree on an interpretation of claims within that organization (even if the claims are not publically standardized). The operator should, however, limit processing of that claim only to the trust domains with which the meaning has been agreed.
+
+#### 7.7.4 Veracity of claims
+
+Just because a trust domain authority asserts a claim does not guarantee the claim is true. When choosing whether to accept particular claims from particular trust domains, operators may wish to conduct audits, ask for evidence, or otherwise investigate the processes by which the trust domain makes the claim.
+
+Imagine a hypothetical claim involving whether the peer workload is in a PCI-compliant (Payment Card Industry security specification, a standard for processing credit card data) environment. If we are going to share credit card data with workloads that are PCI-compliant, we might ask to see evidence of a 3rd party PCI audit and ask how the trust domain authority determines which workloads to issue the claim to. 
+
+If we determine we cannot trust core SPIFFE claims, like the SPIFFE ID, from a particular trust domain, it makes little sense to allow federation at all.  Non-required claims can be filtered by control planes and/or workloads if they are not trusted.
```

Comment:
```suggestion
If an SVID is issued by an untrusted or unknown authority, federation with the foreign trust domain has not been established and any claims contained within the SVID should be treated as untrusted. Once federation has been established with a foreign trust domain and SVIDs of that domain can be authenticated, control plane operators must determine whether to trust any claims which are not required by the SPIFFE specifications or underlying document format (e.g. X-509 or JWT specifications); non-standardized claims may be filtered by control planes and/or workloads if the issuer is not trusted to assert the specific claims.
```

Reply:
I think this rewording glosses over the point that it's actually worth asking yourself whether a foreign trust domain can be trusted to faithfully assert the SPIFFE-required claims, and that if the answer is no, you should not federate at all.